src/components/LeftNavBar/LeftNavBar.tsx

```diff
@@ -4,11 +4,11 @@ export const LeftNavBar = () => {
   return (
     <>
       <section className={styles.contentLeftSite}>
-        <li>HOME</li>
-        <li>FAQS</li>
-        <li>BLOG</li>
-        <li>CONTACT US</li>
-        <li>BUY THEME</li>
+        <button>HOME</button>
```

Comment:
You shouldn't create some navigation using buttons - button is specyfic tag responsible for sending forms, when you wan't achieve similar look to button - use anchor tag `<a></a>` and style it

Reply:
actually you should use Link component from react router